lib/pg_strutil.c

```diff
@@ -542,7 +546,11 @@ ParseFunction(const char *value, bool argistype)
 	strncpy(buf, value, nextp - value);
 	buf[nextp - value] = '\0';
 
+#if PG_VERSION_NUM >= 160000
```

Comment:
Do we really need the block because stringToQualifiedNameList() can be called in [pgut-be.h](https://github.com/ossc-db/pg_bulkload/pull/155/files#diff-8c56469965326364e1239cf5528d8a83ae27c992e3a3b97204479d2cfcab100d)?

Reply:
stringToQualifiedNameList() in pgut-be.h is valid for pg8.3 and below, but later versions refer to stringToQualifiedNameList() in utils/regproc.h.